src/components/ModelizerNavigationBar.vue

```diff
@@ -104,6 +105,7 @@ async function save() {
 
   await gitGlobalSave(project.value)
     .then(() => {
+      FileEvent.GlobalSaveFilesEvent.next();
```

Comment:
- why a new commit for that and not add this modification inside `Update ModelizerNavigationBar` commit ?
- add test to check if `GlobalSaveFilesEvent` is emitted ?

Reply:
done